test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -46,14 +48,37 @@ def _run_ort(
 
 
 def _run_test_with_fx_to_onnx_exporter_reference_runtime(
-    model, input_args, rtol: float = 1e-3, atol: float = 1e-7, opset_version: int = 17
+    model: Union[torch.nn.Module, Callable],
+    input_args,
+    rtol: float = 1e-3,
+    atol: float = 1e-7,
+    opset_version: int = 17,
+    **input_kwargs,
 ):
+    # Feed args and kwargs into exporter.
+    # Note that exporter should flatten kwargs into positional args the exported model;
+    # since ONNX doesn't represent kwargs.
     onnx_model = fx_onnx.export_without_kwargs(
```

Comment:
The api says "export_without_kwargs" Is it still accurate naming?

Reply:
I think it can be changed to `export_by_flatten_args_and_kwargs`.

Reply:
Final name and doc string:
```python
@_beartype.beartype
def export_after_normalizing_args_and_kwargs(
    fn: Union[torch.nn.Module, Callable],
    opset_version,
    *args,
    use_binary_format: bool = True,
    **kwargs,
) -> Union["onnx.ModelProto", bytes]:
    """Export an nn.Module or a callable to ONNX.

    This traces the given nn.Module or a callable into FX graph and then
    and exports it to ONNX by calling `_export`. Notice that ONNX does
    not represent keyword arguments, so `args` and `kwargs are normalized by
    calling `inspect.Signature.bind` and `BoundArgument.apply_defaults`
    in the beginning.

    Args:
        fn: a function or nn.Module or a callable to be exported to ONNX.
        opset_version: the opset version to export the model to. E.g., 14.
        args: the positional arguments to pass to the function.
        use_binary_format: whether to return the ONNX model in binary format.
            If False, onnx.ModelProto will be returned. If False, the byte array
            generated by onnx.ModelProto.SerializeToString is returned.
        kwargs: the keyword arguments to pass to the function.

    Returns:
        ONNX model in binary format or onnx.ModelProto. To select return type,
        use use_binary_format argument.
    """
```